src/com/proj/A.java

```diff
@@ -0,0 +1,38 @@
+package com.proj;
+
+public class A {
+    //Fields
+    int a = 3;
+    static int j;
+    int i = 0;
+
+    //Not statistic block
```

Comment:
@KYehor, do not use the comment, everybody knows java constructions

Reply:
My first Commit, what, please read some article on how to name the commit

Reply:
Where did you show console output? It can be readme file
Create a few class that extend each other and show in console initialization order od static, no-static blocks and constructors

A, B extends A, C extends B